api/src/client/ApiFormationClient.test.ts

```diff
@@ -1223,6 +1223,514 @@ describe("ApiFormationClient", () => {
+                    Zipcode: members[0].addressZipCode,
+                    Country: "US",
+                  },
+                },
+                {
+                  Name: members[1].name,
+                  Location: {
+                    Address1: members[1].addressLine1,
+                    Address2: members[1].addressLine2,
+                    City: members[1].addressCity,
+                    State: members[1].addressState?.name,
+                    Zipcode: members[1].addressZipCode,
+                    Country: "US",
+                  },
+                },
+                {
+                  Name: members[2].name,
+                  Location: {
+                    Address1: members[2].addressLine1,
+                    Address2: members[2].addressLine2,
+                    City: members[2].addressCity,
+                    State: members[2].addressState?.name,
+                    Zipcode: members[2].addressZipCode,
+                    Country: "US",
+                  },
+                },
+              ],
+              Signers: [
+                {
+                  Name: incorporators[0].name,
+                  Title: "General Partner",
+                  Signed: true,
+                },
+              ],
+              ContactFirstName: formationFormData.contactFirstName,
+              ContactLastName: formationFormData.contactLastName,
+              ContactPhoneNumber: formationFormData.contactPhoneNumber,
+            },
+          },
+          { headers: { "Content-Type": "application/json" } }
+        );
+      });
+
+      it("posts to the endpoint with the api formation object – when nonprofit provisions are IN_FORM", async () => {
+        const stubResponse = generateApiResponse({});
+        mockAxios.post.mockResolvedValue({ data: stubResponse });
+        const incorporators = [
+          generateFormationIncorporator({
+            name: "faraz",
+            signature: true,
+            title: "General Partner",
+            ...generateFormationNJAddress({}),
+          }),
+        ];
+
+        const members = [
+          generateFormationMember({}),
+          generateFormationMember({}),
+          generateFormationMember({}),
+        ];
+        const formationFormData = generateFormationFormData(
+          {
+            agentNumberOrManual: "MANUAL_ENTRY",
+            hasNonprofitBoardMembers: true,
+            nonprofitBoardMemberQualificationsSpecified: "IN_FORM",
+            nonprofitBoardMemberQualificationsTerms: "LOREM IPSUM DOLOR SIT AMET",
+            nonprofitBoardMemberRightsSpecified: "IN_FORM",
+            nonprofitBoardMemberRightsTerms: "LOREM IPSUM DOLOR SIT AMET",
+            nonprofitTrusteesMethodSpecified: "IN_FORM",
+            nonprofitTrusteesMethodTerms: "LOREM IPSUM DOLOR SIT AMET",
+            nonprofitAssetDistributionSpecified: "IN_FORM",
+            nonprofitAssetDistributionTerms: "LOREM IPSUM DOLOR SIT AMET",
```

Comment:
[pebble] In order for this to be a test that gives confidence, these LOREM IPSUM should all be four different strings.

If I coded it with a mistaken copy-paste error so that whatever the user types into `nonprofitBoardMemberQualificationsTerms` is what is sent to the API for as the value for all four fields `MemberTerms`, `MemberClassPermissions`, etc - this test would still pass. 

Explicitly expecting each field to be used in its respective API field would give much more confidence that this is working as expected.